tpu/templates/tpu_film_estimator/trainer.ipynb

```diff
@@ -0,0 +1,384 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copyright 2018 Google LLC  \n",
```

Comment:
This is great work, I see that there is reference a paper, can we add some detail around this paper in layman terms? ML problem/Dataset ?
Example:
https://colab.sandbox.google.com/github/tensorflow/tpu/blob/master/tools/colab/bert_finetuning_with_cloud_tpus.ipynb#scrollTo=uu2dQ_TId-uH

Thanks

Reply:
added a little context on FiLM.